plop-templates/component-controller.js.hbs

```diff
@@ -0,0 +1,9 @@
+import angular from 'angular';
+
+class {{properCase name}}Controller {
+  constructor() {}
+}
+
+angular.module('portainer.{{module}}')
+  .controller('{{properCase name}}Controller', {{properCase name}}Controller);
+export default {{properCase name}}Controller;
```

Comment:
Would rather invert those 2 lines to keep ES6 code together and appart from AngularJS code (the only ref to angular will be the last line of the file + the import).

+ in the future we should be able to leverage only the ES6 class on `component.js` file, so iirc we won't need to register the class as Controller in AngularJS.

Reply:
why in the feature? we can do that right now. export the controller and import it in the component file, no need to register it